### PR TITLE
confy: 0.6.5 -> 0.7.0

### PR DESCRIPTION
**Pull request by michaelgrahamevans**
## Description of changes

[Confy 0.7.0 release notes](https://git.sr.ht/~fabrixxm/confy/refs/0.7.0)

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/confy/default.nix [hunk 1]
```diff
@@ -16,14 +16,15 @@
 
 stdenv.mkDerivation rec {
   pname = "confy";
-  version = "0.6.5";
+  version = "0.7.0";
 
   src = fetchurl {
     url = "https://git.sr.ht/~fabrixxm/confy/archive/${version}.tar.gz";
-    sha256 = "sha256-zfuwOZBSGQzJUc36M6C5wSHarLbPFqayQVFo+WbVo7k=";
+    sha256 = "sha256-ZvIzgCMDfUW9g+qmY0ZIUoEF0PeVmINjOPwilmFzWDk=";
```
Comment (AndersonTorres) [inline pkgs/applications/misc/confy/default.nix:23]:
```suggestion
    hash = "sha256-ZvIzgCMDfUW9g+qmY0ZIUoEF0PeVmINjOPwilmFzWDk=";
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Review by AndersonTorres — APPROVED** [review]

**Review by ryantm — APPROVED** [review]
Code changes look good

**Comment by doronbehar** [comment]
I'm experiencing hash differences here, on `staging-next`:

```
0.7.0.tar.gz> trying https://git.sr.ht/~fabrixxm/confy/archive/0.7.0.tar.gz
0.7.0.tar.gz>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
0.7.0.tar.gz>                                  Dload  Upload   Total   Spent    Left  Speed
0.7.0.tar.gz> 100 93571    0 93571    0     0   105k      0 --:--:-- --:--:-- --:--:--  105k
error: hash mismatch in fixed-output derivation '/nix/store/zk23m71yzy0y711y1lx4r34fwgkqc99c-0.7.0.tar.gz.drv':
         specified: sha256-ZvIzgCMDfUW9g+qmY0ZIUoEF0PeVmINjOPwilmFzWDk=
            got:    sha256-qOU/rPj7ZhwWafQ3Vx7K41fdAyxz6jMhFUqU3mH5CHo=
error: 1 dependencies of derivation '/nix/store/8q9picd6yil72j9bjv1xgr5bw2fq02zx-confy-0.7.0.drv' failed to build
```

**Comment by yu-re-ka** [comment]
should probably be fetchzip

**Comment by doronbehar** [comment]
Done already: https://github.com/NixOS/nixpkgs/pull/246584

**Comment by michaelgrahamevans** [comment]
Thank you :pray:

**Pull request merged**